File: src/generator/vue-admin/crawler/crawler/src/main/java/com/jasmine/crawler/pojo/entity/Url.java
/*----------------------------------------------------------------------------
 * Jasmine code generator, a tool to build web crud application,with spring-
 * boot, mybatis, mysql,swagger,spring-security.
 * Generated at 2020-7-31 9:09:23 PM 
 * All rights reserved by fuanlei(email:767550758@qq.com) since 2019
 *---------------------------------------------------------------------------*/
package com.jasmine.crawler.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("url info")
@Getter
@Setter
public class Url  {

    @ApiModelProperty("")
    private String url;

    @ApiModelProperty("")
    private Integer depth;

    @ApiModelProperty("the site that the url belong")
    private Integer downSystemSiteId;

    @ApiModelProperty("")
    private Integer urlType;

    @ApiModelProperty("the related url")
    private String referUrl;

    @ApiModelProperty("current retry count of url")
    private Integer crawlCount;

    @ApiModelProperty("crawl status")
    private Integer urlStatus;

    @ApiModelProperty("last crawl  time")
    private String lastCrawlTime;

    @ApiModelProperty("")
    private String queryString;

    @ApiModelProperty("")
    private Date createTime;

}

File: src/generator/vue-admin/crawler/db/url.js
let url = {
	name: "url",
	description: "url info",
	columns: {
		url: {
			name: "url",
			description: "",
			rawName: "url",
			nullable: false,
			type: {
				name: "varchar",
				length: 200
			}
		},
		depth: {
			name: "depth",
			description: "",
			rawName: "depth",
			nullable: false,
			type: {
				name: "smallint",
				length: 6
			}
		},
		downSystemSiteId: {
			name: "downSystemSiteId",
			description: "the site that the url belong",
			rawName: "down_system_site_id",
			nullable: true,
			type: {
				name: "int",
				length: 11
			}
		},
		urlType: {
			name: "urlType",
			description: "",
			rawName: "url_type",
			nullable: false,
			type: {
				name: "smallint",
				length: 6
			}
		},
		referUrl: {
			name: "referUrl",
			description: "the related url",
			rawName: "refer_url",
			nullable: true,
			type: {
				name: "varchar",
				length: 45
			}
		},
		crawlCount: {
			name: "crawlCount",
			description: "current retry count of url",
			rawName: "crawl_count",
			nullable: true,
			type: {
				name: "int",
				length: 11
			}
		},
		urlStatus: {
			name: "urlStatus",
			description: "crawl status",
			rawName: "url_status",
			nullable: true,
			type: {
				name: "smallint",
				length: 6
			}
		},
		lastCrawlTime: {
			name: "lastCrawlTime",
			description: "last crawl  time",
			rawName: "last_crawl_time",
			nullable: true,
			type: {
				name: "varchar",
				length: 45
			}
		},
		queryString: {
			name: "queryString",
			description: "",
			rawName: "query_string",
			nullable: true,
			type: {
				name: "varchar",
				length: 300
			}
		},
		createTime: {
			name: "createTime",
			description: "",
			defaultValue: "",
			rawName: "create_time",
			nullable: false,
			type: {
				name: "timestamp"
			}
		}
	},
	rawName: "url",
	primaryColumn: "url",
	nameColumn: {
		name: "url",
		score: 1.5992898649205665,
		content: "url",
		weight: 1,
		totalTokenCount: 1
	}
};

let columnsArray = [
	"url",
	"depth",
	"downSystemSiteId",
	"urlType",
	"referUrl",
	"crawlCount",
	"urlStatus",
	"lastCrawlTime",
	"queryString",
	"createTime"
];

function create() {
  return {};
}

url.columnsArray=columnsArray;
url.create=create;
module.exports.url=url;